test_rclcpp/CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ if(BUILD_TESTING)
       TIMEOUT 15)
     custom_gtest(gtest_executor
       "test/test_executor.cpp"
-      TIMEOUT 30)
+      TIMEOUT 60)
```

Comment:
Why is this being changed? The PR doesn't mention it. Is it supposed to help with the flakiness of the test?